src/react-components/media-browser.js

```diff
@@ -387,23 +390,32 @@ class MediaBrowser extends Component {
 
           <div className={styles.body}>
             <div className={classNames({ [styles.tiles]: true, [styles.tilesVariable]: isVariableWidth })}>
-              {urlSource === "avatars" && (
+              {(urlSource === "avatars" || urlSource === "scenes") && (
                 <div
-                  style={{ width: `${createAvatarWidth}px`, height: `${createAvatarHeight}px` }}
-                  className={classNames(styles.tile, styles.createAvatar)}
+                  style={{ width: `${createTileWidth}px`, height: `${createTileHeight}px` }}
+                  className={classNames(styles.tile, styles.createTile)}
                 >
-                  <a
-                    onClick={e => {
-                      e.preventDefault();
-                      window.dispatchEvent(new CustomEvent("action_create_avatar"));
-                    }}
-                    className={styles.tileLink}
-                  >
-                    <div className={styles.tileContent}>
-                      <FontAwesomeIcon icon={faPlus} />
-                      <FormattedMessage id={`media-browser.create-avatar`} />
-                    </div>
-                  </a>
+                  {urlSource === "scenes" ? (
+                    <a href="/spoke/projects/new" rel="noopener noreferrer" target="_blank" className={styles.tileLink}>
+                      <div className={styles.tileContent}>
+                        <FontAwesomeIcon icon={faPlus} />
+                        <FormattedMessage id="media-browser.create-scene" />
+                      </div>
+                    </a>
+                  ) : (
+                    <a
+                      onClick={e => {
+                        e.preventDefault();
+                        window.dispatchEvent(new CustomEvent("action_create_avatar"));
```

Comment:
This should probably be another action?

Reply:
Looks okay to me. I think you just misread the diff.